src/runtime_regex.cpp

```
@@ -10,14 +10,17 @@
 #include <memory>
 
+
+namespace ry {
+
 namespace {
 
 // ============================================================
 // NFA
 // ============================================================
 
 struct NFAState {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: kind

(cppcoreguidelines-pro-type-member-init)